addons/skin.estouchy/language/resource.language.en_gb/strings.po

```diff
@@ -179,7 +179,7 @@ msgstr ""
 #empty strings from id 31304 to 31308
 
 msgctxt "#31309"
-msgid "Memory Used:"
+msgid "System Memory Used:"
```

Comment:
Since your touching it, mind choosing used or usage? For consistency sake...
Also, please don't capitalize the second word.